### PR TITLE
fix error when contact has no email

### PR DESCRIPTION
**Pull request by caplod**
If you delete the email in a contact and then try to send an email to a group with that contact you get an error:

```TypeError: t.contacts[e].email is null```

here I add a check if the contact has an email.

### CODE REVIEW

File: src/components/AppNavigation/GroupNavigationItem.vue [hunk 1]
```diff
@@ -154,6 +154,10 @@ export default {
 		emailGroup(group) {
 			const emails = []
 			group.contacts.forEach(key => {
+        // break if contact has no email
+				if (this.contacts[key].email == null) {
```
Comment (ChristophWurst) [inline src/components/AppNavigation/GroupNavigationItem.vue:158]: is the prop `null` or undefined? let's use strong comparison with `===`

File: src/components/AppNavigation/GroupNavigationItem.vue [hunk 2]
```diff
@@ -154,6 +154,10 @@ export default {
 		emailGroup(group) {
 			const emails = []
 			group.contacts.forEach(key => {
```
Comment (ChristophWurst) [inline src/components/AppNavigation/GroupNavigationItem.vue:156]: nitpick: you can add a `.filter` before `forEach` to filter out these contacts

Comment (caplod) [reply src/components/AppNavigation/GroupNavigationItem.vue]: ok, I added a filter method

**Review by ChristophWurst — APPROVED** [review]
:sunglasses: 

If you can squash your commits into one, then we can :shipit:

**Pull request merged**